#
# Message for the state of the amtec powercube pan/tilt unit. 
#
Header  header         # header
uint32  state          # module state
float64 position       # module angle in radians
float64 velocity       # module velocity in radians/s